hazelcast/src/main/java/com/hazelcast/map/impl/LazyMapEntry.java

```diff
@@ -43,8 +44,7 @@
  * @see com.hazelcast.map.impl.operation.EntryOperation#createMapEntry(Data, Object)
  */
 
-public class LazyMapEntry extends CachedQueryEntry implements Serializable, IdentifiedDataSerializable {
-    private static final long serialVersionUID = 0L;
+public class LazyMapEntry extends CachedQueryEntry implements IdentifiedDataSerializable {
```

Comment:
I see that Serializable has been removed. Are you sure that can't cause problems?